app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -142,3 +142,22 @@
     }
   }
 }
+
+.callout-side-arrow{ //this works best if it's a class on a svg element.
+  position: absolute;
+  z-index: 1;
+  top: 0;
+
+  @include breakpoint(small down){
+    top: 100%;
+    left: 0%;
+    height: 30%;
+    background-color: $white;
+  }
+}
+
+.mobile-arrow-margin{
```

Comment:
Hmmm what is the element above? Is there a reason you can't add it to the arrow itself?